meta-lmp-base/recipes-containers/docker/docker-ce_%.bbappend

```diff
@@ -14,13 +14,15 @@ SRC_URI:append = " \
 
 DOCKER_MAX_CONCURRENT_DOWNLOADS ?= "3"
 DOCKER_MAX_DOWNLOAD_ATTEMPTS ?= "5"
+DOCKER_DAEMON_JSON_CUSTOM ?= ""
```

Comment:
The addiction of the new DOCKER_DAEMON_JSON_CUSTOM variable can be skipped in more recent version of the `nvidia-container-toolkit` and the json snip can be installed to `${datadir}/containers/oci/hooks.d`.
AFAIK not supported yet on meta-tegra.

Reply:
Yeah, the one provided by the nvidia tegra bsp is quite behind the version used by x86. Tried to update some of the components but then got several issues.